src/Catty/ViewController/Continue&New/MaintainObject/MaintainLooks/LookImageViewController.m

```diff
@@ -153,7 +153,7 @@ - (void)addPaintedImage:(UIImage *)image andPath:(NSString *)path
 
     [imageData writeToFile:self.imagePath atomically:YES];
     
-    NSString *imageDirPath = [[self.spriteObject projectPath] stringByAppendingString:kProjectImagesDirName];
+    NSString *imageDirPath = [[self.spriteObject projectPath] stringByAppendingFormat:@"%@/%@", [Util defaultSceneNameForSceneNumber:1], kProjectImagesDirName];
```

Comment:
The look does not necessarily be located in the first scene (also the first scene can have a different name).